blueoil/templates/blueoil-config.tpl.yml

```diff
@@ -11,6 +11,8 @@ dataset:
 trainer:
   batch_size: {{ batch_size }}
   epochs: {{ training_epochs }}
+  training_learning_rate: {{ training_learning_rate_setting }}
```

Comment:
I think `learning_rate_setting`  is better name than `training_learning_rate` .

Reply:
Also, plz write comment to easy change for blueoil user.
like this: 
https://github.com/blue-oil/blueoil/blob/2a6604d7a17516aa25df8de527c516c571a95fc9/blueoil/templates/blueoil-config.tpl.yml#L1-L2

Reply:
Sorry, I was mislead.

As already discussed,
The name of `learning_rate_setting` is good. 
The comment of `learning_rate_setting` is like below.

from
```
training_learning_rate: {{ training_learning_rate_setting }}
```

to
```
# supported 'learning_rate_setting' is 'tune1', 'tune2', 'tune3', 'fixed'.
#   'tune1' is 2 times decay, learning rate reduce to 1/10 on epoch/2 and epoch-1. 
#   'tune2' is 3 times decay, learning rate reduce to 1/10 on epoch/3 and epoch*2/3 and epoch-1. 
#   'tune3' is warmup and 3 times decay, warmup learning rate 1/1000 in 1 epoch, then train same as 'tune2'. 
#   'fixed' is constant learning rate.
learning_rate_setting: {{ training_learning_rate_setting }}
```

Reply:
Thank you. I rewrite like this.

Reply:
Thanks 👍